### PR TITLE
Weather - App

### PR DESCRIPTION
**Pull request by karthykarthick**
A weather app built by Js using The concepts that i understand  It fetches your current location and display the weather
- You can Search your favourite City

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,93 @@
...
+});
+
+const getWeather = async (city) => {
+  try {
+    const response = await fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=90582edac32da819f08413ad8ebbcb45`
+    );
+
+    const weatherData = await response.json();
+    console.log(weatherData);
```
Comment (blackpintz) [inline src/index.js:28]: Remove this `console.log`.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Invalid

**Hello team, I have to mark this pull request as invalid. Please re-open the old pull request and submit that one instead, after working on the requested changes.**

### Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi)**

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUESTED🛑🛑**

Hello @karthykarthick 
Great implementation of the project. I love how easy you made it look.
However let's make some changes to make the project more professional.

[REQUIRED] When you enter a city, the background should change
[REQUIRED] Remove the empty index.js file in components folder 
[REQUIRED] in src/index.js, on line 20, 24, use ES6. Instead of var use let or const.

Make the requested changes and submit for another review👍

**Comment by karthykarthick** [comment]
When you enter a city, the background should change it is not specified in requirements

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @karthykarthick  ! :wave:

#### I really liked the simplicity of your project! Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

#### You may merge, good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**